.github/workflows/2_bump_versions.yml

```diff
@@ -0,0 +1,55 @@
+name: Bump versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      version:
+        description: 'New Prisma CLI version'
+        required: true
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Install Semver # Needed to use semver in scripts
+        run: npm install
```

Comment:
Does this really install Semver? Right now this would only install https://github.com/prisma/language-tools/blob/simplify_CI/package.json which does not include it or do I misunderstand?

Reply:
Where would this actually be used?

Reply:
Good point, I was missing the dependency in the root package.json. It's used in  'bump_extension_version.js'